tests/unit/challenges/test_views.py

```diff
@@ -2510,13 +2514,31 @@ def setUp(self):
             content_type="application/zip",
         )
 
-    def test_create_challenge_using_zip_file_when_zip_file_is_not_uploaded(
-        self
-    ):
         self.url = reverse_lazy(
             "challenges:create_challenge_using_zip_file",
             kwargs={"challenge_host_team_pk": self.challenge_host_team.pk},
         )
+
+        self.BASE_TEMP_LOCATION = tempfile.mkdtemp()
+        self.base_path = join(settings.BASE_DIR, 'tests', 'unit', 'challenges', 'data')
+
+        self.path_to_annotation = join(self.base_path, 'annotations')
+        self.path_to_original_yaml_file = join(self.base_path, 'challenge_config.yaml')
+        self.path_to_altered_yaml = join(self.BASE_TEMP_LOCATION, "altered_yaml_file.yaml")
+        self.path_to_sample_file = join(self.BASE_TEMP_LOCATION, 'sample.txt')
+        self.path_to_eval_script_zip = join(self.base_path, 'evaluation_script.zip')
+
+        self.yaml_file = open(join(self.base_path, 'challenge_config.yaml'))
+        self.yaml_dict = yaml.safe_load(self.yaml_file)
+        self.copy_dict = copy.deepcopy(self.yaml_dict)
+
+        self.alt_file = open(self.path_to_altered_yaml, 'w+')
+        self.alt_file.write("Sample")
```

Comment:
This is not a valid yaml? Please create an invalid yaml alongside with valid yaml.